src/SFML/Window/Context.cpp

```diff
@@ -27,8 +27,8 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/Context.hpp>
 #include <SFML/Window/GlContext.hpp>
-#include <SFML/System/ThreadLocalPtr.hpp>
 #include <SFML/System/Err.hpp>
+#include <cstddef>
```

Comment:
What's this header included for?

Reply:
Seems like this is a remnant of something else, it is not necessary. I will remove it.